pkg/apis/datadoghq/v1alpha1/const.go

```diff
@@ -6,7 +6,7 @@
 package v1alpha1
 
 const (
-	// AgentDeploymentNameLabelKey label key use to link a Resource to a DatadogAgentDeployment
+	// AgentDeploymentNameLabelKey label key use to link a Resource to a DatadogAgent
 	AgentDeploymentNameLabelKey = "agentdeployment.datadoghq.com/name"
```

Comment:
should we consider renaming these as well? `agent.datadoghq.com/name`

Reply:
good idea